### ISSUE TITLE
How to solve?I can't start training using main.py.

### ISSUE THREAD

**Issue by Like2021** [fundamentalvision/Deformable-DETR#161]
**Thanks for your great work!**
**Cuz single gpu,I used** 

`python main.py --coco_path "../coco/images" --batch_size=2 --num_workers=8 --output_dir="output" --start_epoch 0 --resume=r50_deformable_detr-checkpoint.pth `

**but like this,prompt me:**

Warning: (hack) args.override_resumed_lr_drop is set to True, so args.lr_drop would override lr_drop in resumed lr_scheduler.
/home/admin1/anaconda3/envs/deformable_detr/lib/python3.8/site-packages/torch/optim/lr_scheduler.py:154: UserWarning: The epoch parameter in `scheduler.step()` was not necessary and is being deprecated where possible. Please use `scheduler.step()` to step the scheduler. During the deprecation, if epoch is different from None, the closed form is used instead of the new chainable form, where available. Please open an issue if you are unable to replicate your use case: https://github.com/pytorch/pytorch/issues/new/choose.
  warnings.warn(EPOCH_DEPRECATION_WARNING, UserWarning)
Start training
Training time 0:00:00

**Then program is over.No more tips.**

**I don't know why.**

**Comment by jeiva2000** [comment]
As you are using a pre-training model, this line redefines from which time you will start training:

args.start_epoch = checkpoint['epoch'] + 1

So the argument epochs must be may

**Comment by Like2021** [comment]
got it!
i knew the reason
thanks


> 当您使用预训练模型时，此行重新定义了您将开始训练的时间：
> 
> args.start_epoch = 检查点['epoch'] + 1
> 
> 所以论点时期必须是可能的
